.github/workflows/parachain.yml

```diff
@@ -333,6 +333,9 @@ jobs:
         run: |
           cd test
           pnpm typegen
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: diesel_cli
```

Comment:
What is this for?

Reply:
This is the ORM app we use for our indexer DB. To setup the env you need to migrate the DBs to the latest schema in `client/indexer-db/migrations`